source/frontend/src/features/documents/documentDetail/DocumentDetailView.tsx

```diff
@@ -40,31 +34,32 @@ export const DocumentDetailView: React.FunctionComponent<
       {hasClaim(Claims.DOCUMENT_VIEW) && (
         <>
           <DocumentDetailHeader document={props.document} />
-          <StyledGreySection>
-            <Row className="pb-3">
-              <Col className="text-left">
-                <StyledHeader>
-                  <StyledH2>Document information</StyledH2>
-                  <TooltipIcon
-                    toolTipId="documentInfoToolTip"
-                    innerClassName="documentInfoToolTip"
-                    toolTip="Information you provided here will be searchable"
-                  ></TooltipIcon>
-                </StyledHeader>
-              </Col>
-              {hasClaim(Claims.DOCUMENT_EDIT) && (
-                <Col xs="2">
-                  {' '}
-                  <LinkButton
-                    onClick={() => {
-                      props.setIsEditable(true);
-                    }}
-                  >
-                    <FaEdit />
-                  </LinkButton>
-                </Col>
-              )}
-            </Row>
+
+          <Section
```

Comment:
This seems duplicated from the DocumentDetailHeader

Reply:
Not sure what you mean.